backend/app.js

```diff
@@ -7,17 +7,12 @@ import swaggerJsDoc from 'swagger-jsdoc';
 
 import { StatusCodes } from 'http-status-codes';
 import errorController from './controllers/errorController.js';
-import AppError from './utils/appError.js';
-
-import ajvValidator from './utils/ajvValidator.js';
-import queueSchema from './schemas/queueSchema.js';
-import resolutionSchema from './schemas/resolutionSchema.js';
+import { AppError } from './utils/errorClasses.js';
```

Comment:
For what this import?

Reply:
for sending the 404 error if user is trying to access a non-existent route